activestorage/test/controllers/blobs/redirect_controller_test.rb

```diff
@@ -68,7 +68,7 @@ class ActiveStorage::Blobs::RedirectControllerWithOpenRedirectTest < ActionDispa
     end
   end
 
-  if SERVICE_CONFIGURATIONS[:azure]
+  if false # SERVICE_CONFIGURATIONS[:azure]
```

Comment:
Maybe add a comment as of why?

Reply:
Also rather than `false`, something like `!ENV['CI']`, so that if someone get their hands on the credentials they can run the suite locally.

Reply:
Yes, that makes sense.

Reply:
I've added a comment and moved the disabling to the intialization of the `SERVICE_CONFIGURATIONS` as suggested by @dhh.